.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+*   nik@victoriametrics.com @valyala @tenmozes
```

Comment:
Why one owner is email, and others are github usernames? I'd vote to stay consistent.